code_of_conduct.md

```diff
@@ -0,0 +1,7 @@
+Winter Tech Forum Code of Conduct
+=================================
+
+1. Don't be an asshole.
```

Comment:
I like the idea of the simple version, but I don't think this is good enough. Reason being that most cases of harassment are unintentional. Otherwise put: just because you think you're not being an asshole doesn't necessarily translate to someone not feeling harassed by you. Pycon does this well:

```
Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
```

What's missing from this short version: definitions of what kinds of behavior count as harassment, and what the consequence is for someone who engages in harassment. Pycon does this here:

```
If a participant engages in behavior that violates this code of conduct, the conference organizers may take any action they deem appropriate, including warning the offender or expulsion from the conference with no refund.
```